[debug] [2018-04-20T00:43:01.731Z] ----------------------------------------------------------------------
[debug] [2018-04-20T00:43:01.733Z] Command:       /usr/local/bin/node /Users/kimmyeongsu/.npm-global/bin/firebase serve
[debug] [2018-04-20T00:43:01.733Z] CLI Version:   3.18.4
[debug] [2018-04-20T00:43:01.733Z] Platform:      darwin
[debug] [2018-04-20T00:43:01.733Z] Node Version:  v8.11.1
[debug] [2018-04-20T00:43:01.734Z] Time:          Thu Apr 19 2018 20:43:01 GMT-0400 (EDT)
[debug] [2018-04-20T00:43:01.734Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-04-20T00:43:01.741Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-04-20T00:43:01.741Z] > authorizing via signed-in user
[debug] [2018-04-20T00:43:01.743Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/cs252-lab6-58dac  
  
 Thu Apr 19 2018 20:43:01 GMT-0400 (EDT)
[debug] [2018-04-20T00:43:01.961Z] <<< HTTP RESPONSE 200
[debug] [2018-04-20T00:43:01.961Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/cs252-lab6-58dac/tokens  
  
 Thu Apr 19 2018 20:43:01 GMT-0400 (EDT)
[debug] [2018-04-20T00:43:02.238Z] <<< HTTP RESPONSE 200
[debug] [2018-04-20T00:43:02.239Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/cs252-lab6-58dac  
  
 Thu Apr 19 2018 20:43:02 GMT-0400 (EDT)
[debug] [2018-04-20T00:43:02.409Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/kimmyeongsu/Desktop/project6/StraightEye-Lab6'...
[info] 
[debug] [2018-04-20T00:43:02.920Z] Error: EACCES: permission denied, mkdir '/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/lib/functionsEmulator.js:111:21)
    at Object.start (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:631:23)
    at Function.map (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9546:14)
    at _serve (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/Users/kimmyeongsu/.npm-global/lib/node_modules/firebase-tools/commands/serve.js:48:12)
[error] 
[error] Error: An unexpected error has occurred.
